src/main/resources/application.properties

```diff
@@ -1,7 +1,14 @@
 spring.datasource.url=jdbc:postgresql://localhost:5432/event_organizer
 spring.datasource.username=dachi
 spring.datasource.password=password
+
 spring.jpa.hibernate.ddl-auto=update
 spring.jpa.show-sql=true
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
-spring.jpa.properties.hibernate.format_sql=true
\ No newline at end of file
+spring.jpa.properties.hibernate.format_sql=true
+
+spring.flyway.enabled=true
+spring.flyway.url=jdbc:postgresql://localhost:5432/event_organizer
+spring.flyway.user=dachi
+spring.flyway.password=password
```

Comment:
Passwords should not be visible in git repository. You may have also received github's security warning email.
You can pass passwords as environment variables, like 
`spring.flyway.password=${DB_PASSWORD}`
and run application with parameters DB_PASSWORD=password

Reply:
https://www.jetbrains.com/help/objc/add-environment-variables-and-program-arguments.html